test/org/apache/tomcat/websocket/TestWsWebSocketContainer.java

```diff
@@ -779,6 +779,104 @@ public void testSessionExpirySession() throws Exception {
         Assert.assertEquals(0, getOpenCount(setA));
     }
 
+    @Test
+    public void testSessionExpiryOnUserPropertyReadIdleTimeout() throws Exception {
+        final String readIdleTimeoutParameter = org.apache.tomcat.websocket.Constants.WS_READ_IDLE_TIMEOUT;
```

Comment:
I opted to use the Constants class (as it is in the same package) and use the fully qualified name with the Constants from the server package.